manifests/update.pp

```diff
@@ -56,13 +56,27 @@
   } else {
     $_refresh = true
   }
+  $apt_update_successful = @("ASH"/$)
+    try=1
+    while [ \$try -le ${apt::_update['tries']} ]; do
+      timeout ${apt::_update['timeout']} ${apt::provider} update
+      code=\$?
+      [ \$code -eq 0 ] && exit 0
+      [ \$try -eq 1 ] && exit \$code
+      sleep 1
```

Comment:
never reached: if `timeout` succeeds, we exit with `0` 2 lines above, otherwise we exit with `$code` 1 line above.

Reply:
Line 66 is only reached if $try is greater than 1.

Reply:
@smortex

Reply:
(looking again)... Crap; you're right. I'm counting up, not down. Gotta compare it to `${apt::update['tries']}`, not to `1`.

Fixed and re-pushed.